libcxx/modules/std/exception.inc

```diff
@@ -21,6 +21,5 @@ export namespace std {
   using std::terminate;
   using std::terminate_handler;
   using std::throw_with_nested;
-  using std::uncaught_exception;
```

Comment:
This should be
```
#ifdef _LIBCPP_ENABLE_CXX20_REMOVED_UNCAUGHT_EXCEPTION
  using std::uncaught_exception;
#endif
```
instead of an unconditional remove.

Reply:
Done. But currently we haven't made any other C++17/20/23-removed components restorable in standard library modules. Should we do this later?